example.env

```diff
@@ -0,0 +1,4 @@
+MAIL_HOST= sandbox.smtp.mailtrap.io
+MAIL_PORT= 465
+MAIL_USER= 14e06e26e6e95c
+MAIL_PASSWORD= e93f4808aeb665
```

Comment:
In case my comment wasn't clear. We should not have credentials committed in the code